lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -748,7 +749,8 @@ instance Arbitrary PostExternalTransactionData where
     arbitrary = do
         count <- choose (0, 32)
         bytes <- BS.pack <$> replicateM count arbitrary
-        return $ PostExternalTransactionData bytes
+        let hex = convertToBase Base16 bytes
+        return $ PostExternalTransactionData hex
```

Comment:
This seems incorrect The conversion from and to hexadecimal should only happen at the edge of the system, `PostExternalTransactionData` should only contain raw bytes.

Having said that... Is this arbitrary instance used anywhere? We are running JSON roundtrips here, but since the JSON instance has disappeared :thinking: ?

Reply:
agree here, we do not need conversion to hex anymore as we dealt with this at the api level. removed it.

nevertheless, we need arbitrary here as it is used in test : 
```
 describe "pointless tests to trigger coverage for record accessors"
```